src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredArray = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter(Boolean);
+  const styleObject = {};
+
+  for (const element of filteredArray) {
+    const [property, value] = element.split(':').map((part) => part.trim());
```

Comment:
The split operation on `element` by colon is correct for separating property and value. However, note that handling multiple colons within a value is not required according to the additional prompt instructions.